packages/salesforcedx-vscode-lwc/test/vscode-integration/activation.test.ts

```diff
@@ -0,0 +1,236 @@
+import { fail } from 'assert';
+import * as path from 'path';
+import { assert, SinonStub, stub } from 'sinon';
+import * as vscode from 'vscode';
+
+describe('Activation of the LWC Language Server', () => {
+  it('Language Server should start when an LWC js file is opened', async () => {
+    const lwcExtension: vscode.Extension<any> = vscode.extensions.getExtension(
```

Comment:
Fetching the extension should be pulled out to a before/beforeEach block.  This is duplicated across multiple test cases.